spec/spec.md

```diff
@@ -28,24 +28,27 @@ Participate:
 ------------------------------------
 
 ## Abstract
+  
+The JWT VC Presentation Profile defines a standardized way to interoperably exchange [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
```

Comment:
Why original text is not enough? I think "defines standardized way" is misleading - it is not a new specification as you point out below.

```suggestion
The JWT VC Presentation Profile defines a set of specifications to enable the interoperable presentation of [[ref:Verifiable Credentials]] (VCs) between digital [[ref: Wallets]] and credential [[ref: Verifiers]].
```